output.cc

```diff
@@ -2526,6 +2526,9 @@ void printfinaloutput() {
       log_write(LOG_PLAIN, "OS detection performed. Please report any incorrect results at https://nmap.org/submit/ .\n");
     else if (o.servicescan)
       log_write(LOG_PLAIN, "Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .\n");
+    else if (o.udpscan && o.defeat_icmp_ratelimit)
+      log_write(LOG_PLAIN, "WARNING: Some ports marked closed|filtered may actually be open. For more accurate results, do not use --defeat-icmp-ratelimit .\n");
```

Comment:
I tried to use `"Note: Some ports marked ..."`. It did not catch attention as it should have done, so I changed to the initial Daniel's suggestion, warning.

Should I have used `error("WARNING: Some ports marked...")` or current version is ok?